bin/start-dbs.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+if [ -z "$CI" ]; then
+  MONGO_CONTAINER_NAME="mongodb_c"
+  docker rm -f $MONGO_CONTAINER_NAME
```

Comment:
is it a dup to remove it again in the start file? I notice that the same cmd is called in `stop-dbs.sh`.

Reply:
It's just to be sure that such containers are removed. We can call `./stop-dbs.sh` instead.

Reply:
> Not really, I added stop-dbs.sh so that I can shut down these containers manually for local tests.

Oh I see 👌